pkg/types/openstack/platform.go

```diff
@@ -62,6 +62,10 @@ type Platform struct {
 	// +optional
 	ClusterOSImage string `json:"clusterOSImage,omitempty"`
 
+	// UseHwQEMUAgentForClusterOSImage indicates that the property "hw_qemu_guest_agent=yes" should be added to the Glance ClusterOSImage
+	// +optional
+	UseHwQEMUAgentForClusterOSImage bool `json:"useHwQEMUAgentForClusterOSImage,omitempty`
```

Comment:
```suggestion
	UseHWQEMUAgentForClusterOSImage bool `json:"useHwQEMUAgentForClusterOSImage,omitempty`
```

also maybe create a *ClusterOSImageProperties struct if you think there will be more properties. and enforce that it be nil when clusterOSImage is set to pre existing glance image.

also needs more clearer documentation when this field is used and not, like is user gave us glance image we probably can't change or add these properties.